### ISSUE TITLE
Scroll bar doesn't work in the Game Manager / Logs

### ISSUE THREAD

**Issue by rocco27** [OpenGamePanel/OGP-Website#138]
Panel and Agent up-to-dated. I think this is very old bug.

Open a game server, use rcon command (example CoD4  -> Log -> use 'status' rcon command. I always have the scroll on the right side, but if the result is too long from the game server, then it is not possible to scroll down to read all.

**Comment by Obada8** [comment]
It's work, but the scroll bar not go up because the log refreshed every 4 second. You can click on [+] button and use scroll bar.

**Comment by own3mall** [comment]
Can you please provide a screenshot?

**Comment by own3mall** [comment]
@rocco27 did @Obada8 suggestion work?

**Comment by rocco27** [comment]
Sorry, it is here: http://i.imgur.com/mtfKIOR.png

**Comment by AlexanderCurl** [comment]
No, that scroll doesn't work with the latest source. And only displays cca. 10 lines only. If it's longer the panel just ignore it.

**Comment by Adjokip** [comment]
The scrollbars have always been like that.

There's no height set on the output, and overflow:scroll is set on the output div... so scrollbars will be visible for both vertical and horizontal sides.

The following line should be changed to fix it: [gamemanager/rcon.php#130](https://github.com/OpenGamePanel/OGP-Website/blob/master/modules/gamemanager/rcon.php#L130) and change the style attribute of the XMP tag to overflow:auto, likely also wrap htmlentities around $response.

The issue seems to be some output is cut off (the last bot name and address), which doesn't seem to be related to OGP. I can get a maximum of 132 lines / 4024 characters returned from the console before stuff is cut off.

**Comment by AlexanderCurl** [comment]
Thats fine, however sometimes the 132 lines not enough. For example the CoD4 dvarlist more than 700 lines.

**Comment by own3mall** [comment]
So just download the log file then, and open it with a text editor?

**Comment by AlexanderCurl** [comment]
Well, ok, I can, but the error still present, and it would be nice if its work, because some command just not going into the log.

**Comment by Adjokip** [comment]
I've fixed this on the PRs I referenced above. The Perl KKRcon class was the cause, limiting what was returned and thus cutting off the output.

Another PR has been submitted to this repo to remove the vertical scrollbar on the console output, as it'll never be usable unless a height property is set on the element.

One it's been submitted, try `dvarlist` ... it works fine with `cvarlist` on a Source-based gameserver.
